src/Sylius/Component/Core/OrderProcessing/OrderTaxesProcessor.php

```diff
@@ -38,14 +40,19 @@ final class OrderTaxesProcessor implements OrderProcessorInterface
     /** @var PrioritizedServiceRegistryInterface */
     private $strategyRegistry;
 
+    /** @var TaxationAddressResolverInterface|null */
+    private $taxationAddressResolver;
+
     public function __construct(
         ZoneProviderInterface $defaultTaxZoneProvider,
         ZoneMatcherInterface $zoneMatcher,
-        PrioritizedServiceRegistryInterface $strategyRegistry
+        PrioritizedServiceRegistryInterface $strategyRegistry,
+        ?TaxationAddressResolverInterface $taxationAddressResolver
     ) {
         $this->defaultTaxZoneProvider = $defaultTaxZoneProvider;
         $this->zoneMatcher = $zoneMatcher;
         $this->strategyRegistry = $strategyRegistry;
+        $this->taxationAddressResolver = $taxationAddressResolver;
```

Comment:
We should add deprecation notice if one would omit to inject this service. Something like this: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php#L71-L73